samsungctl/remote_websocket.py

```diff
@@ -0,0 +1,72 @@
+import base64
+import json
+import logging
+import socket
+import time
+import websocket
+
+from . import exceptions
+
+class RemoteWebsocket():
+    """Object for remote control connection."""
+
+    def __init__(self, config):
+        if not config["port"]:
+            config["port"] = 8001;
+
+        """Make a new connection."""
+        self.connection = websocket.WebSocket()
+        self.connection.connect("ws://%s:%d/api/v2/channels/samsung.remote.control?name=%s" % (config["host"], config["port"], self._serialize_string(config["name"])))
```

Comment:
Prefer `format` over `%`. Also, this line is too long.

Perhaps something like this:
```python
URL_FORMAT = "ws://{}:{}/api/v2/channels/samsung.remote.control?name={}"

...

self.connection.connect(URL_FORMAT.format(config["host"], config["port"],
                                          self._serialize_string(config["name"])))
```